webpack.config.js

```diff
@@ -1,22 +1,38 @@
 const path = require("path");
 const CopyWebpackPlugin = require("copy-webpack-plugin");
 const { CleanWebpackPlugin } = require('clean-webpack-plugin');
+const ImageminPlugin = require("imagemin-webpack");
 const webpack = require("webpack");
 
 let config = {
     context: path.resolve(__dirname, "src"),
     entry: "./app.js",
     output: {
         filename: "bundle.js",
-        path: path.resolve(__dirname, "dist"),
+        path: path.resolve(__dirname, "dist")
     },
     plugins: [
         new CleanWebpackPlugin(),
         new CopyWebpackPlugin([{
             from: "**/*",
             to: ".",
             ignore: ['*.js']
-        }])
+        }]),
+        new ImageminPlugin({
+            bail: false,
+            cache: false,
+            imageminOptions: {
+                plugins: [
+                    ["svgo", {
+                        plugins: [
+                            {
+                                removeViewBox: false
```

Comment:
```suggestion
                                removeComments: false,
                                removeViewBox: false
```

Reply:
This will ensure that the license is kept.

Reply:
This is a good change